examples/jsm/loaders/GLTFLoader.js

```diff
@@ -2662,7 +2662,7 @@ class GLTFParser {
 		let isFirefox = false;
 		let firefoxVersion = - 1;
 
-		if ( typeof navigator !== 'undefined' ) {
+		if ( typeof navigator !== 'undefined' && typeof navigator.userAgent !== 'undefined' ) {
```

Comment:
Can you make this change in `KTX2Loader.detectSupport()` as well?

https://github.com/mrdoob/three.js/blob/96dbfa62827c08976b256d5e0c68a71e4cce0352/examples/jsm/loaders/KTX2Loader.js#L254-L255